addons/l10n_cl/models/account_move.py

```diff
@@ -27,20 +27,23 @@ def _get_l10n_latam_documents_domain(self):
         domain = super()._get_l10n_latam_documents_domain()
         if (self.journal_id.l10n_latam_use_documents and
                 self.journal_id.company_id.country_id == self.env.ref('base.cl')):
+            domain += [('active', '=', True)]
             if self.type in ['in_invoice', 'in_refund']:
                 if self.partner_id.l10n_cl_sii_taxpayer_type == '2':
                     domain += [('code', '=', '71')]
                 return domain
             document_type_ids = self.journal_id.l10n_cl_sequence_ids.mapped('l10n_latam_document_type_id').ids
             domain += [('id', 'in', document_type_ids)]
             if self.partner_id.l10n_cl_sii_taxpayer_type == '3':
-                domain += [('code', 'in', ['35', '38', '39', '41'])]
+                domain += [('code', 'in', {'35', '38', '39', '41'})]
```

Comment:
Still in Odoo we normally prefer [], just to be uniform.

Reply:
@Danisan I think this the cause of one of the errors Rubi encountered.  Our views don't accept domains with {}.

Reply:
Fixed by @suv-bmya at 712aeab2